samples/bluetooth/peripheral/src/main.c

```diff
@@ -287,6 +284,33 @@ static void bt_ready(void)
 		settings_load();
 	}
 
+	err = bt_gatt_get_svc_uuid_data(BT_DATA_UUID16_ALL, &ad[1],
+					uuid16_buf, sizeof(uuid16_buf));
+	if (err != 0) {
+		printk("Could not encode all UUID16 data: %d\n", err);
+
+		err = bt_gatt_get_svc_uuid_data(BT_DATA_UUID16_SOME, &ad[1],
+						uuid16_buf, sizeof(uuid16_buf));
+		if (err != 0) {
+			printk("Could not encode UUID16 data: %d\n", err);
+			return;
+		}
+	}
+
+	err = bt_gatt_get_svc_uuid_data(BT_DATA_UUID128_ALL, &ad[2],
+					uuid128_buf, sizeof(uuid128_buf));
+	if (err != 0) {
+		printk("Could not encode UUID128 data: %d\n", err);
+
+		err = bt_gatt_get_svc_uuid_data(BT_DATA_UUID128_SOME, &ad[2],
+						uuid128_buf,
+						sizeof(uuid128_buf));
+		if (err != 0) {
+			printk("Could not encode UUID128 data: %d\n", err);
+			return;
+		}
+	}
+
```

Comment:
Is this really a good sample to choose to demonstrate the API? You end up adding many more lines and complexity compared to the original code. In fact, do you actually have a good use case for this API, where you end up with more readable code after the change?

Reply:
Yes, I have a better sample (unicast_audio_server) in mind that I can use instead :)